examples/bluetooth/nimble/ble_phy/phy_prph/tutorial/Ble_Phy_Peripheral_Example_Walkthrough.md

```diff
@@ -0,0 +1,412 @@
+# BLE Peripheral PHY Example Walkthrough
+
+## Introduction
+
+In this tutorial, the ble_phy peripheral example code for the espressif chipsets with BLE5.0 support is reviewed. This example aims at understanding how to establish connections on preferred PHY and changing LE PHY once the connection is established. The code implements a BLE Peripheral PHY, which establishes a connection on LE 1M PHY and switches to LE 2M PHY/CODED once the connection is established.
```

Comment:
"This example aims at understanding how to establish connections on preferred PHY and changing LE PHY once the connection is established"

To 

"This example aims at understanding how to accept incoming connections on preferred PHY and process the request to change LE PHY once the connection is established"